.readme-partials.yaml

```diff
@@ -94,22 +94,22 @@ custom_content: |
   imports at the top of your file:
   
   ```java
-  import com.google.cloud.Page;
+  import com.google.api.gax.paging.Page;
   import com.google.cloud.logging.LogEntry;
   import com.google.cloud.logging.Logging.EntryListOption;
   ```
   
   Then, to list the log entries, use the following code:
 
   ``` java
-  Page<LogEntry> entries = logging.listLogEntries(
-      EntryListOption.filter("logName=projects/" + options.getProjectId() + "/logs/test-log"));
-  Iterator<LogEntry> entryIterator = entries.iterateAll().iterator();
-  while (entryIterator.hasNext()) {
-    System.out.println(entryIterator.next());
+  Page<LogEntry> entries = logging.listLogEntries(EntryListOption.filter(logFilter));
```

Comment:
From unfamiliar user's perspective it's not clear here where logFilter is coming from and what should be its value

Reply:
Good catch :+1: , fix it right away